apis/duck/ABOUT.md

```diff
@@ -302,7 +302,11 @@ is used with duck types:
                 ...
         }
         logger := c.Logger.Named("Revisions")
-        impl := controller.NewImpl(c, logger, "Revisions")
+        impl := controller.NewContext(
+          ctx,
+          c,
+          controller.ControllerOptions{WorkQueueName: "Revisions", Logger: logger},
```

Comment:
is this change valid? Seems like a merge conflict.

Reply:
Yup now the new version of the pkg have this format on the imp initialization

And for the comment below, for now we could stick to the same cloudevents validations